docs/Customizing Look and Functionality/Customizing Help Prompts/index.md

```diff
@@ -0,0 +1,131 @@
+
+@![iOS]
+##### Objective-C
+```objc
+SDLSetGlobalProperties *setGlobals = [[SDLSetGlobalProperties alloc] init];
+setGlobals.vrHelpTitle = <#Custom help title string such as: "What Can I Say?"#>
+
+// Set up the menu items
+SDLVRHelpItem *item1 = [[SDLVRHelpItem alloc] initWithText:<#Help item name such as "Show Artists"#> image: <#A previously uploaded image or nil#> position: 1];
+SDLVRHelpItem *item2 = [[SDLVRHelpItem alloc] initWithText:<#Help item name such as "Shuffle All"#> image: <#A previously uploaded image or nil#> position: 2];
+setGlobals.vrHelp = @[item1, item2];
+
+[self.sdlManager sendRequest:setGlobals withResponseHandler:^(SDLRPCRequest *request, SDLRPCResponse *response, NSError *error) {
+    if (error != nil) {
+        // Something went wrong
+    }
+
+    // The help prompt is updated
+}];
+```
+
+##### Swift
+```swift
+let setGlobals = SDLSetGlobalProperties()
+setGlobals.vrHelpTitle = <#Custom help title string such as: "What Can I Say?"#>
+
+// Set up the menu items
+let item1 = SDLVRHelpItem(text:<#Help item name such as "Show Artists"#>, image: <#A previously uploaded image or nil#>, position: 1)
+let item2 = SDLVRHelpItem(text:<#Help item name such as "Shuffle All"#>, image: <#A previously uploaded image or nil#>, position: 2)
+setGlobals.vrHelp = [item1, item2];
+
+sdlManager.send(request: setGlobals) { (request, response, error) in
+    if error != nil {
+        // Something went wrong
+    }
+
+    // The help prompt is updated
+}
+```
+!@
+
+@![android, javaSE, javaEE]
+```java
+// TODO
+```
+!@
+
+## Help Prompt
+In SDL, the user is able to request assistance with using your app by saying "Help" on head units that support voice recognition. This will display the help menu discussed above, but it can also speak a custom spoken text-to-speech response to the user.
+
+@![iOS]
+##### Objective-C
+```objc
+SDLSetGlobalProperties *setGlobals = [[SDLSetGlobalProperties alloc] init];
+setGlobals.helpPrompt = SDLTTSChunk *response = [SDLTTSChunk textChunksFromString:<#Your custom help prompt#>];
+
+[self.sdlManager sendRequest:setGlobals withResponseHandler:^(SDLRPCRequest *request, SDLRPCResponse *response, NSError *error) {
+    if (error != nil) {
+        // Something went wrong
+    }
+
+    // The help prompt is updated
+}];
+```
+
+##### Swift
+```swift
+let setGlobals = SDLSetGlobalProperties()
+setGlobals.helpPrompt = SDLTTSChunk.textChunks(from: <#Your custom help prompt#>)
+
+sdlManager.send(request: setGlobals) { (request, response, error) in
+    if error != nil {
+        // Something went wrong
+    }
+
+    // The help prompt is updated
+}
+```
+!@
+
+@![android, javaSE, javaEE]
+```java
+// TODO
+```
+!@
+
+## Timeout Prompt
```

Comment:
Should probably add another section "Clearing the Help Menu" with instructions on how to remove the custom help menu.